cor_graphical/code/cor_data_preprocess.R

```diff
@@ -9,6 +9,8 @@ library(dplyr)
 library(tidyverse)
 library(stringr)
 dat.mock <- read.csv(here("..", "data_clean", data_name))
+dat.mock$wt[is.na(dat.mock$wt)] <- 0
```

Comment:
@KenLi93 @yiwenlu 
Can you confirm that it is OK to set the missing weights to 0 in down stream code?